react/Loader/Loader.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import styles from './Loader.less';
+import classnames from 'classnames';
 
-const Loader = () => (
-  <div className={styles.root}>
-    <div className={styles.ball} />
-    <div className={styles.ball} />
-    <div className={styles.ball} />
-  </div>
-);
+export default function Loader({ inline }) {
+  return (
+    <div className={classnames(styles.root, { [styles.inline]: inline })}>
+      <div className={styles.ball} />
+      <div className={styles.ball} />
+      <div className={styles.ball} />
+    </div>
+  );
+}
 
-export default Loader;
+Loader.propTypes = {
+  inline: PropTypes.bool.isRequired
```

Comment:
There's no need for it to be required prop as we set default value

Reply:
Yep good catch I will remove.